src/test/regress/sql/upgrade_distributed_function_before.sql

```diff
@@ -0,0 +1,39 @@
+CREATE SCHEMA upgrade_distributed_function_before;
```

Comment:
nit; I would remove the `_before` suffix from the schema. It is in the name of the test only to signal when in the lifecycle of the test it is called. What the set of two files are testing is `upgrade_distributed_function`.

That said, it is not super important for the functionality.